templates/web/base/main_nav_items.html

```diff
@@ -0,0 +1,38 @@
+[%~ PROCESS navitem uri='/' label=loc('Report a problem') attrs='class="report-a-problem-btn"' ~%]
+
+[%~ IF c.user_exists ~%]
+    [%~ PROCESS navitem uri='/my' label=loc('Your account') ~%]
+[%~ ELSE ~%]
+    [%~ PROCESS navitem uri='/auth' label=loc('Sign in') ~%]
+[%~ END ~%]
+
+[%~ IF c.user_exists AND c.user.has_body_permission_to('planned_reports') ~%]
+    [%~ PROCESS navitem uri='/my/planned' label=loc('Shortlist') ~%]
+[%~ END ~%]
+
+[%~
+  IF c.user_exists AND c.user.categories.size;
+    categories = c.user.categories.join(",") | uri;
+    suffix = "?filter_category=" _ categories;
+  END;
+
+  IF body_name;
+     body_name = body_name | uri;
+     suffix = "/${body_name}${suffix}";
+  END;
+
+  PROCESS navitem uri='/reports' label=loc('All reports');
+~%]
+
+[%~
+  IF pc;
+    pc_uri = pc | uri;
+    suffix = "/list?pc=" _ pc_uri;
+  END;
+
+  PROCESS navitem uri='/alert' label=loc('Local alerts');
+~%]
+
+[%~ PROCESS navitem uri='/faq' label=loc('Help') ~%]
+
+[%~ PROCESS navitem uri='/privacy' label=loc('Privacy') liattrs='class="nav-menu__item--privacy"' ~%]
```

Comment:
If it's currently not linked anywhere, I assume it's not a page that will have been adapted for that cobrand, and so that should presumably be maintained?